package.json

```diff
@@ -38,8 +38,10 @@
   },
   "dependencies": {
     "cli-spinner": "0.2.10",
+    "colorette": "^1.1.0",
     "commander": "3.0.2",
     "dockerode": "3.0.2",
+    "log-update": "^3.3.0",
     "loglevel": "1.6.4",
     "node-emoji": "1.10.0",
     "progress": "2.0.3",
```

Comment:
I noticed a couple of packages in here that were not being used: `progress`, `rxjs`.. just thought I'd tell you about it; if you thought they could be removed, I could remove them!

Reply:
If they are not needed, you can remove them :)